README.md

```diff
@@ -91,16 +106,18 @@ nextflow run bam2fastq.nf \
 
 ## Environment and Execution
 
-Environment and execution are two relatively indepently varying parameters that you can select by choosing a profile combination:
+[ the significance of this section is not entirely clear, especially to people unexperienced with Nextflow ]
```

Comment:
I understand, but this section is not meant to be for those with no experience. I think, the only reasonable thing to do is to maybe make a better reference to the Nextflow docs.